assets/svelte/transforms/Index.svelte

```diff
@@ -59,60 +81,127 @@
           data-phx-link="redirect"
           data-phx-link-state="push"
         >
-          <Button>Create your first function</Button>
+          <Button size="sm">
+            <FilePlus class="h-4 w-4 mr-2" />
+            Create your first function
+          </Button>
         </a>
       </div>
     </div>
   {:else}
-    <Table.Root>
+    <Table.Root class="text-sm">
       <Table.Header>
-        <Table.Row>
+        <Table.Row class="text-xs text-gray-500 uppercase tracking-wide">
           <Table.Head>Name</Table.Head>
           <Table.Head>Type</Table.Head>
-          <Table.Head>Function</Table.Head>
+          <Table.Head>Consumers</Table.Head>
           <Table.Head>Created</Table.Head>
-          <Table.Head>Last updated</Table.Head>
+          <Table.Head>Updated</Table.Head>
+          <Table.Head>Edit</Table.Head>
+          <Table.Head>Delete</Table.Head>
```

Comment:
I propose using text links instead of icons, like this comparable:

![CleanShot 2025-05-17 at 09 08 16@2x](https://github.com/user-attachments/assets/7897ff18-3c3c-484e-a880-0bc06e2f3875)

So, this would be:

```suggestion
          <Table.Head></Table.Head>
          <Table.Head></Table.Head>
```